src/ts/downloaders/AccountImageDownloader.ts

```diff
@@ -21,13 +20,15 @@ export class AccountImageDownloader extends Downloader {
      * Download the account image
      */
     private static async downloadContent(): Promise<void> {
-        const response = await makeRequest(location.href);
-        const pictureURL = response.owner.profile_pic_url_hd;
-        const accountName = response.owner.username;
+        const response = (await (await fetch(`${location.href}?__a=1`)).json()).graphql;
+        const pictureURL = response.user.profile_pic_url_hd;
+        const accountName = response.user.username;
+        const timestamp = new Date().getTime() / 1000; // There is no timestamp for profile pictures, use the current time
 
         const downloadMessage: DownloadMessage = {
             imageURL: [pictureURL],
-            accountName,
+            accountName: accountName,
```

Comment:
The second accountName can be left empty. Normally the tslint should highlight this